docs/api.rst

```diff
@@ -62,7 +62,7 @@ API Reference
             if ``verify_exp``, ``verify_iat``, and ``verify_nbf`` respectively
             is set to ``True``).
 
-    :param Iterable audience: optional, the value for ``verify_aud`` check
+    :param Union[str, Iterable] audience: optional, the value for ``verify_aud`` check
```

Comment:
There is no other usage of `Union` in the doc, tell me if you think there is a better way to do it.
(ps: I did not use `Optional` to keep it consistent with the lines bellow)